src/config/section/iam.js

```diff
@@ -176,6 +176,17 @@ export default {
         title: 'Accounts',
         param: 'domainid'
       }],
+      tabs: [
+        {
+          name: 'details',
+          component: () => import('@/components/view/DetailsTab.vue')
+        }, {
+          name: 'settings',
+          permission: ['listConfigurations'],
+          component: () => import('@/views/iam/TemplateDomainSettings.vue')
+        }
+      ],
+      treeView: true,
```

Comment:
This will require refactoring, however, I'll accept it.

Reply:
I think it is similar to https://github.com/apache/cloudstack-primate/pull/53#discussion_r353568566